### PR TITLE
Qa photos

### PR DESCRIPTION
**Pull request by Laweeza**
I ended up using aws-sdk and multer to figure out the broken images.

- Most of the troubleshooting was on aws.amazon.com and permissions with my s3 bucket. 
- Images are now visible upon upload!
image

image

*sry I merged the changes into main already but would still like you guys to know as an fyi lol

### CODE REVIEW

**Review by logan-tm — APPROVED** [review]

File: server/controllers/answers.js [hunk 1]
```diff
@@ -15,17 +15,17 @@ module.exports ={
...
-      console.log('error', err)
-      res.status(400).send(err)
-    })
-  },
+  // post: (req, res) => {
+  //   const {body, name, email, photos} = req.body
+  //   console.log('params', photos, req.body)
+  //   models.answers.post(req.params, req.body)
+  //   .then(({rows}) => models.photos.post(rows[0], req.body))
+  //   .then((result) => res.status(201).send(result))
```
Comment (Laweeza) [inline server/controllers/answers.js:23]: I didn't want to get rid of my old answers post controller code yet, just commented it out for now.

File: server/app.js [hunk 2]
```diff
@@ -20,57 +27,58 @@ const storage = multer.diskStorage({
...
+
 const upload = multer({ storage: storage });
 
 app.use(logger('dev'))
 app.use(express.json())
 app.use(express.urlencoded({extended: true}))
 app.use(cors())
+
+
 app.use('/api/qa', router)
```
Comment (Laweeza) [inline server/app.js:40]: I basically uprooted my answers post controller route and did the logic in the server lol. Probably will reformat later if I have time. All other api's are still in my router file.

File: server/app.js [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
 
-const PHOTO_UPLOAD_FOLDER = path.join('/Users/louisa/HR/HR2021/SDC/Atelier', '/client/UploadedPhotos');
 
-app.use(express.static(PHOTO_UPLOAD_FOLDER));
+const staticAssets = path.resolve(__dirname.substring(0, __dirname.length - 17), 'Atelier/client/dist')
+app.use(express.static(staticAssets))
+
+
+const PHOTO_UPLOAD_FOLDER = path.join(__dirname.substring(0, __dirname.length - 17), 'Atelier/client/UploadedPhotos');
+app.use(express.static(PHOTO_UPLOAD_FOLDER))
```
Comment (Laweeza) [inline server/app.js:19]: On the front end I noticed that the FEC team had a photo folder so I just also re-directed my photo uploads to this folder as well. Not a good idea to keep the actual photos in the backend anyways due to greater potential for bloating and slowing down backend processes, specifically the database.

File: package.json [hunk 4]
```diff
@@ -31,6 +36,7 @@
     "express": "^4.17.1",
     "morgan": "^1.10.0",
     "multer": "^1.4.3",
+    "multer-s3": "^2.10.0",
```
Comment (Laweeza) [inline package.json:39]: Didn't end up using multer-s3, but plan on using if I have time to refactor and make my files less ugly.

**Pull request merged**